nowcasting_dataset/data_sources/satellite/satellite_data_source.py

```diff
@@ -55,6 +56,8 @@ def _get_time_slice(self, t0_dt: pd.Timestamp) -> xr.DataArray:
         start_dt = self._get_start_dt(t0_dt)
         end_dt = self._get_end_dt(t0_dt)
         data = self.data.sel(time=slice(start_dt, end_dt))
+        data = data.astype(np.int16)
```

Comment:
What would you think about _not_ including this line (and hence `nowcasting_dataset` would just propagate the satellite data type from the incoming Zarr)?

On the one hand, keeping this here makes it more explicit.

On the other hand, you could imagine a situation where someone updates `Satip` to use a different dtype for the satellite Zarr (e.g. int8 or float16) and we might forget to update the dtype here, which might result in weird bugs.   It might be nice to only have to change the dtype in one place in the code, and to have clearly defined responsibilities between the different repos?  (e.g. we could declare that `nowcasting_dataset` never changes the numerical values: it just chops up the data)

Reply:
probably best to remove it for now - so we just default what is in `Satip`

Reply:
SGTM!